Resources/frontend/js/jquery.adyen-confirm-order.js

```diff
@@ -134,10 +134,13 @@
         handlePaymentDataIdentifyShopper: function (data) {
             var me = this;
             var paymentData = data.paymentData;
+
             $(me.opts.placeOrderSelector).parent().append('<div id="AdyenIdentifyShopperThreeDS2"/>');
             me.adyenCheckout
                 .create('threeDS2DeviceFingerprint', {
-                    fingerprintToken: data.authentication['threeds2.fingerprintToken'],
+                    token: data.authentication['threeds2.fingerprintToken'],
+                    useOriginalFlow: true,
```

Comment:
@acampos1916 
The new flow would require a clientKey, so for upgrade I left the original flow active.
Should we update to the new  flow with the ClientKey property for web components?

Similar for *threeDS2Challenge*